src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/RoadSegmentSurfaceAttributesToZipArchiveWriter.cs

```diff
@@ -28,9 +29,8 @@ public RoadSegmentSurfaceAttributesToZipArchiveWriter(RecyclableMemoryStreamMana
         if (request == null) throw new ArgumentNullException(nameof(request));
         if (context == null) throw new ArgumentNullException(nameof(context));
 
-        var attributes = await context.RoadSegmentSurfaceAttributes
-            .InsideContour(request.Contour)
-            .ToListAsync(cancellationToken);
+        var attributes = await context.RoadSegmentSurfaceAttributes.ToListWithPolygonials(request.Contour, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken);
```

Comment:
use a line-separated method chain (like what it was)

Reply:
Done